samples/snippets/visualbasic/VS_Snippets_Wpf/WPFWithWFAndDatabinding/VisualBasic/WPFWithWFAndDatabinding/WPFWithWFAndDatabinding.vbproj

```diff
@@ -60,28 +33,11 @@
       <SubType>Code</SubType>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
-    <Import Include="Microsoft.VisualBasic" />
-    <Import Include="System" />
-    <Import Include="System.Collections" />
-    <Import Include="System.Diagnostics" />
-    <Import Include="System.Windows" />
```

Comment:
WDYT about these WPF-/WinForms-specific imports? Seems that the WindowsDesktop SDK doesn't automatically add these.

Reply:
Ping @BillWagner @adegeo

Reply:
@NextTurn Sorry for the delay. I don't know why they were there. It's compiling fine, so I assume this is good to go?

Reply:
With these removed, `Imports` statements need to be added to separate files. (See `Imports System.Windows` in `App.xaml.vb`.)

Reply:
Then they should be added back in?

Reply:
Just a matter of style. I don't program in VB so I don't know which would be preferred.

Reply:
Now I think it's better to use explicit imports in source files because they are easier to look up.y